index.jsx

```diff
@@ -2,18 +2,22 @@
 
 var React = require('react');
 var ReactRouter = require('react-router');
+var contains = require('lodash.contains')
 
 var Router = ReactRouter;
 var Route = ReactRouter.Route;
 var RouteHandler = ReactRouter.RouteHandler;
 var Link = ReactRouter.Link;
 
+
+
 var Breadcrumbs = React.createClass({
     propTypes: {
         separator: React.PropTypes.string,
         displayMissing: React.PropTypes.string,
         displayName: React.PropTypes.string,
-        breadcrumbName: React.PropTypes.string
+        breadcrumbName: React.PropTypes.string,
+        excludes: React.ProptTypes.arrayOf(React.propTypes.string)
```

Comment:
This one needs to be 

             excludes: React.PropTypes.arrayOf(React.PropTypes.string)